components/FeatureHighlight/src/private/MaterialFeatureHighlightStrings.h

```diff
@@ -1,5 +1,6 @@
 // GENERATED CODE: DO NOT EDIT BY HAND
-// GENERATED FROM: components/FeatureHighlight/src/MaterialFeatureHighlight.bundle/Resources/en.lproj/MaterialFeatureHighlight.strings
```

Comment:
This code was generated, we likely want to omit generated code from the formatter.